main.py

```diff
@@ -18,8 +18,7 @@
     conn = ut.oracle_connect(db_ip, db_user, db_pass, db_sid)
     sql = ut.read_query(sql_path)
     site_results = ut.collect_data(sql, conn, site)
-
-    print(site_results)
-
+    results = results.append(site_results)

```

Comment:
why you set the results var here?

append func already adding the item to the list

Reply:
this is the syntax to add the SQL results to the main Data-Frame